src/compile/mod.rs

```diff
@@ -54,6 +54,8 @@ pub struct CompileStruct {
     pub consts_file: Option<String>,
     #[clap(short, long)]
     pub release_build: bool,
+    #[clap(short, long)]
+    pub no_builtins: bool,
```

Comment:
Can the compiler figure out whether to link in the builtins, and just include the builtins that are actually used by the program?  Seems like this could be better than making the programmer guess.

Reply:
This would be significantly more work, and the immediate benefit of even the manual option is large, so I do not believe it would be beneficial to add automatic functionality to this PR.